static/compiler.js

```diff
@@ -106,47 +150,69 @@ function Compiler(domRoot, origFilters, windowLocalPrefix, onChangeCallback, lan
             break;
     }
 
+    // Set up the editor's window
+    // TODO : understand why [0] is mandatory here
```

Comment:
the domRoot.find() returns (effectively) an array of all matching elements. jquery doesn't mind if you use arrays: it applies whatever changes you want to all the elements in the array. However, CodeMirror needs a single DOM node to work with. That's why the [0] is used; to pull the one (and only one) match to this find() query and pass that to CodeMirror.